spec/NewTwitchApi/Resources/ChannelPointsApiSpec.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace spec\NewTwitchApi\Resources;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use PhpSpec\ObjectBehavior;
+use Psr\Http\Message\ResponseInterface;
+
+class ChannelPointsApiSpec extends ObjectBehavior
+{
+    function let(Client $guzzleClient)
+    {
+        $this->beConstructedWith($guzzleClient);
+    }
+
+    function it_should_custom_reward(Client $guzzleClient, Response $response)
```

Comment:
I'd rather this be named `it_should_get_custom_reward` or whatever is similarly appropriate. The idea behind spec names is that they should basically be sentences. "It should custom reward." Not quite a fully sentence, since it's missing a verb. "It should get custom reward." Makes more sense to me. We're describing the behavior in human language. :-) (Same for the other specs below)